ExploreASL_Master.m

```diff
@@ -160,7 +160,8 @@
 
     % -----------------------------------------------------------------------------    
     %% Finishing touch
-    fprintf('Many thanks for using ExploreASL, please don''t forget to cite https://pubmed.ncbi.nlm.nih.gov/32526385/\n');
+    fprintf('Many thanks for using <a href="https://github.com/ExploreASL" rel="nofollow">ExploreASL</a>, ');
+    fprintf('please don''t forget to cite <a href="https://pubmed.ncbi.nlm.nih.gov/32526385/" rel="nofollow">https://pubmed.ncbi.nlm.nih.gov/32526385/</a>.\n');

```

Comment:
This works very nice!

Reply:
Thanks 👍  I don't want to add links everywhere, because they can be a bit ugly in the log files, but I thought here it could make sense.